packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx

```diff
@@ -149,15 +155,17 @@ class UtilityMenuItem extends React.Component {
       { 'default-right-inset': !rightInset },
     ]);
 
-    /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-element-to-interactive-role */
+    const roles = isReadOnly ? 'note' : 'button';
+
+    /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-element-to-interactive-role, jsx-a11y/no-noninteractive-tabindex */
     const renderBodyItem = (fill, wrapOnKeyDown, handleSelection) => (
       <li
         {...customProps}
         tabIndex="0"
         key={itemKey}
-        onClick={event => handleSelection(event, itemKey)}
-        onKeyDown={wrapOnKeyDown(itemKey, onKeyDown)}
-        role="button"
+        onClick={!isReadOnly ? event => handleSelection(event, itemKey) : undefined}
+        onKeyDown={!isReadOnly ? wrapOnKeyDown(itemKey, onKeyDown) : undefined}
+        role={roles}
```

Comment:
Nitpick - since we're doing inline conditionals, it would be fine do the same for role
```jsx
role={isReadOnly ? 'note' : 'button'}
```

I don't have much accessibility knowledge, but we may rework how these roles are set. Looks like we we may open a first class prop to configure this: https://github.com/cerner/terra-framework/issues/272. That is out of scope though, just something to ponder

Reply:
Inlining this conditional should be fine. Let's tackle opening the role prop as a first-class prop in a separate PR.